canonical-web-and-design/ubuntu.com
# Redirects /edge to /blog/tag/edge

## Done

- Redirects /edge to /blog/tag/edge

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/edge
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check that http://0.0.0.0:8001/edge redirects to http://0.0.0.0:8001/blog/tag/edge

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4164
